### ISSUE TITLE
Bug: FTP 

### ISSUE THREAD

**Issue by diego83200** [Dolibarr/dolibarr#12471]
# Bug

## Environment
- **Version**: 10.0.3
- **OS**: Windows NT
- **Web server**: Apache/2.4.9 (Win32) OpenSSL/1.0.1g PHP/5.5.12
- **PHP**: apache2handler 5.5.12
- **Database**: MySQL or MariaDB 5.0.45-community-nt
- **URL**: /dolibarr/ftp/index.php?action=download&numero_ftp=3&section=%2F1.Comptes+%26+Supports&file=Comptes+et+Supports+131119.xlsx

Hello

just for tell you that the FTP does not works with an error message in the file index.php, when i want to download any files on any FTthanks for your help

**Comment by futurehousestore** [comment]
Where you have an option to download files in the standard FTP module?

**Comment by diego83200** [comment]
HelloIf you don't have an option in ftp module for downloading files this not a ftp? At the right on the screen you have a icon sheet for downloading files but it works only for pdf xlsx docx files but not for zip files or big files the Connexion freeze and we cannot do anything just reboot the computer Thanks Envoyé depuis mon téléphone Huawei-------- Message original --------De : futurehousestore <notifications@github.com>Date : ven. 22 nov. 2019 à 15:34À : Dolibarr/dolibarr <dolibarr@noreply.github.com>Cc : diego83200 <jclauret@sfr.fr>, Author <author@noreply.github.com>Objet : Re: [Dolibarr/dolibarr] Bug: FTP  (#12471)Where you have an option to download files in the standard FTP module?

**Comment by futurehousestore** [comment]
Ok, I found the option to download the files (which isn't really clear to guess as it doesn't look like a download option lol). The screenshot is from v. 10.0.3 it looks the same. I have tested it and it works fine for me. Maybe it has something to do with your PHP version... I use 7.1.33 and I don't think version 5 is still supported but I may be wrong.
![11](https://user-images.githubusercontent.com/28427270/69470938-50310e80-0d92-11ea-913d-0bb7fe1f6006.JPG)
